### PR TITLE
Sorting links in footnotes sections throughout site

### LINKED ISSUE #183: Sort links in footnotes section of pages
For exapmle, the links at the end of https://raw.githubusercontent.com/etcd-io/website/master/content/docs/next/op-guide/runtime-configuration.md should be sorted. 

> ... sort these link definitions (using your editor's "sort lines" features).

_Originally posted by @chalin in https://github.com/etcd-io/website/pull/174#discussion_r605292505_

### PR DESCRIPTION
**Pull request by nate-double-u**
This PR just sorts links. (And removes one duplicated one).

Fixes #183

### CODE REVIEW

File: content/docs/next/upgrades/upgrading-etcd.md [hunk 1]
```diff
@@ -21,8 +21,8 @@ This section contains documents specific to upgrading etcd clusters and applicat
 
 
 [migrate-apps]: ../op-guide/v2-migration
-[upgrade-cluster]: upgrade_3_0
 [upgrade-3-1]: upgrade_3_1
 [upgrade-3-2]: upgrade_3_2
 [upgrade-3-3]: upgrade_3_3
 [upgrade-3-4]: upgrade_3_4
+[upgrade-cluster]: upgrade_3_0
```
Comment (chalin) [inline content/docs/next/upgrades/upgrading-etcd.md:28]: Hmm, I'd inline these definitions (leaving `migrate-apps`). They don't buy us anything AFAICT. E.g.:

    * [Upgrade etcd from 3.0 to 3.1][upgrade-3-1]

becomes

    * [Upgrade etcd from 3.0 to 3.1](upgrade_3_1)

which is just as readable, and we avoid introducing an extra link definition.

**Review by chalin — APPROVED** [review]
LGTM, aside from one nit (see inlined comment).

File: content/docs/next/upgrades/upgrading-etcd.md (hunk 1 above)
Comment (nate-double-u) [reply content/docs/next/upgrades/upgrading-etcd.md]: Agreed, and updated.

**Pull request merged**